Makefile

```diff
@@ -6,8 +6,8 @@ IS_CI=$(shell [ ! -z "$$CI" ] && echo "1")
 ARCH=arch -$(shell [ $(IS_CI) ] && echo "x86_64" || echo "arm64")
 PIPEFAIL=set -o pipefail
 XCPRETTY=bundle exec xcpretty
-TEST=$(PIPEFAIL) && $(ARCH) xcodebuild test -disableAutomaticPackageResolution -skipPackageUpdates -project StytchDemo/StytchDemo.xcodeproj -scheme StytchCoreTests -sdk
```

Comment:
Is there a reason/benefit to removing the `-disableAutomaticPackageResolution` flag?

Reply:
I was getting errors on this for some reason so I deleted it

Reply:
ok added it back and fixed the ignore file.